src/app/components/import/AttachingModalContent.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Bordered, FileInput, Alert } from 'react-components';
+
+import AttachedFile from './AttachedFile';
+
+const AttachingModalContent = ({ attached, file, onAttach, onClear }) => {
+    return (
+        <>
+            <Alert learnMore="https://protonmail.com/support/knowledge-base/adding-contacts/">
+                {c('Description')
+                    .t`We support importing CSV files from Outlook, Outlook Express, Yahoo! Mail, Hotmail, Eudora and some other apps. We also support importing vCard. (UTF-8 encoding).`}
+            </Alert>
+            <Bordered className="flex">
+                {/* TODO: drag&drop component here. There seems to be no React component for this kind of behavior yet */}
+                {attached ? (
+                    <AttachedFile file={file} iconName="contacts-groups" onClear={onClear} />
+                ) : (
+                    <FileInput className="center" accept=".csv, .vcf" id="import-contacts" onChange={onAttach}>
```

Comment:
multiple?

Reply:
that would allow different files to be imported, which you told me not to do. v3 is not allowing multiple import, and indeed the logic for the components is simpler and not so much is gained from the user perspective. I used to have the logic for it, so probably I can recover it if you want to allow for multiple import

Reply:
:+1: